src/main/resources/templates/browse.html

```diff
@@ -24,7 +24,7 @@
     <script src="/js/browse.js" type="text/javascript"></script>
     <script src="/js/shared.js" type="text/javascript"></script>
 
-    <title>Home</title>
+    <title>Browse</title>
```

Comment:
Changing the name to better suit the HTML page.

Reply:
While you're in here, add some more words to describe the page.  What is being browsed?  If I have a bunch of tabs open, having a bit more context than "Browse" will be good to know the purpose of this page.